pkg/output/printer.go

```diff
@@ -142,11 +138,19 @@ func newPagerWithDefault(c *cli.Context) (io.Writer, func()) {
 	output := OutputOption(outputFlag)
 
 	var defaultPager string
-	if output == Table {
+
+	switch output {
+	case "":
+		defaultPager = string(pager.Stdout)
+	case Table:
+	case Card:
```

Comment:
Why not get the user’s pager (in unix it would default to the PAGER env var)? AFAIK more and less are unix specific

Reply:
it does use user's pager if set

however if not set, then we get to this ^ code. Here is how the pager is selected by priorities https://github.com/temporalio/tctl-kit/pull/20#discussion_r947662093

Reply:
updated semantics (default pager -> suggested pager) to better reflect intention

Reply:
I don't think this will work on windows, we should make sure this is a valid approach for any OS

Reply:
why not? if the pager is not found it falls back to stdout

Reply:
https://github.com/temporalio/tctl-kit/blob/0cc3215aed00101ba2042d78bb4a339e209e3c3a/pkg/pager/pager.go#L50

Reply:
By "not found" do you mean it's not a valid executable?

Reply:
yea